src/System.Data.OleDb/ref/System.Data.OleDb.Manual.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Data.OleDb
+{
+    [System.ComponentModel.TypeConverterAttribute(typeof(OleDbParameter.OleDbParameterConverter))]
```

Comment:
I see, we don't have a great way to handle this, outside of maybe making a seperate run of GenAPI to generate just this file.

Today GenAPI only has the following rules for filters:
https://github.com/dotnet/arcade/blob/b93c96232d63a48e4c1d6c9d95ecbc0fa6823cc2/src/Microsoft.DotNet.GenAPI/Program.cs#L227-L237

Using API list, public types, or all types are mutually exclusive.  We have an intersection filter which lets us subtract from these sets.  We should consider a Union filter that lets us add and is driven by a DocID list.  I'll file an issue.

Reply:
Filed https://github.com/dotnet/arcade/issues/2624